### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sugandhalahoti**
## Background

updating documentation

## Changes

- updated new use-cases from FAQs
- added a section on what is panther and how does it work

## Testing

- How did you test your change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,25 @@
...
 
 Panther works by analyzing security-relevant data generated by your clouds, networks, applications, and hosts to enable threat detection, cloud security, and investigations.
 
 Panther provides flexible Python detection logic, secure and automated deployments within your AWS cloud, and support for popular security logs, commonly used for:
 
-- **Detecting Malicious Behavior:** Analyze logs to identify unauthorized or suspicious activity
-- **Threat Hunting:** Quickly search for matches against indicators of compromise, extracted into common data fields
-- **Achieving Compliance:** Use [built-in policies](https://github.com/panther-labs/panther-analysis) as controls for achieving SOC/PCI/HIPAA compliance
-- **Securing Cloud Resources:** Model security best practices with Python policies and automatically fix misconfigurations with automatic remediation
+- **Detect Unauthorized Access:** Analyze host-based logs to identify unauthorized accesss into systems that could indicate a breach
```
Comment (3nvi) [inline README.md]: typo in `access`

File: README.md [hunk 2]
```diff
@@ -22,16 +22,25 @@
...
+- **Power Your Investigations:** Join data from various sources to determine if a potential compromise has occurred.
+- **Threat Hunting:** Quickly search all of your logs for matches against indicators of compromise by utlilizing Panther's standardized data fields
```
Comment (3nvi) [inline README.md]: We could maybe remove the word `your` since it's not added in the other bullets

File: README.md [hunk 3]
```diff
@@ -22,16 +22,25 @@
...
+- **Achieve Compliance:** Use [built-in policies](https://github.com/panther-labs/panther-analysis) as controls for achieving SOC/PCI/HIPAA compliance
+- **Secure Cloud Resources:** Automatically fix misconfigurations that can cause extreme damage
```
Comment (3nvi) [inline README.md]: Subjective: `extreme damage` seems a bit too much for me :P

File: README.md (hunk 1 above)
Comment (3nvi) [reply README.md]: I would change the phrasing to

` ... to identify unauthorized access that could indicate a system breach`

File: README.md (hunk 2 above)
Comment (3nvi) [reply README.md]: also typo in `utilizing`

File: README.md (hunk 1 above)
Comment (rleighton) [inline README.md]: s/Detect/Detecting (since above it says "used for")

File: README.md [hunk 4]
```diff
@@ -22,16 +22,25 @@
...
 Panther works by analyzing security-relevant data generated by your clouds, networks, applications, and hosts to enable threat detection, cloud security, and investigations.
 
 Panther provides flexible Python detection logic, secure and automated deployments within your AWS cloud, and support for popular security logs, commonly used for:
 
-- **Detecting Malicious Behavior:** Analyze logs to identify unauthorized or suspicious activity
-- **Threat Hunting:** Quickly search for matches against indicators of compromise, extracted into common data fields
-- **Achieving Compliance:** Use [built-in policies](https://github.com/panther-labs/panther-analysis) as controls for achieving SOC/PCI/HIPAA compliance
-- **Securing Cloud Resources:** Model security best practices with Python policies and automatically fix misconfigurations with automatic remediation
+- **Detect Unauthorized Access:** Analyze host-based logs to identify unauthorized accesss into systems that could indicate a breach
+- **Power Your Investigations:** Join data from various sources to determine if a potential compromise has occurred.
```
Comment (rleighton) [inline README.md]: s/Power/Powering (since above it says "used for")

File: README.md [hunk 5]
```diff
@@ -22,16 +22,25 @@
...
+- **Threat Hunting:** Quickly search all of your logs for matches against indicators of compromise by utlilizing Panther's standardized data fields
+- **Achieve Compliance:** Use [built-in policies](https://github.com/panther-labs/panther-analysis) as controls for achieving SOC/PCI/HIPAA compliance
```
Comment (rleighton) [inline README.md]: s/Achieve/Achieving (since above it says "used for")

File: README.md (hunk 3 above)
Comment (rleighton) [inline README.md]: s/Secure/Securing (since above it says "used for")

File: README.md (hunk 1 above)
Comment (sugandhalahoti) [reply README.md]: have corrected both, thank you

File: README.md (hunk 2 above)
Comment (sugandhalahoti) [reply README.md]: corrected both, thank you

File: README.md (hunk 3 above)
Comment (sugandhalahoti) [reply README.md]: changed

File: README.md (hunk 1 above)
Comment (sugandhalahoti) [reply README.md]: corrected

File: README.md (hunk 4 above)
Comment (sugandhalahoti) [reply README.md]: corrected

File: README.md (hunk 5 above)
Comment (sugandhalahoti) [reply README.md]: corrected

File: README.md (hunk 3 above)
Comment (sugandhalahoti) [reply README.md]: corrected

**Pull request merged**